static/include/components/board/board-analogs.md

```diff
@@ -72,6 +66,6 @@ The following properties are available for `analogs`:
 |`name` | string | **Required** | Your name for the analog reader. |
 |`pin`| string | **Required** | The pin number of the ADC's connection pin, wired to the board. This should be labeled as the physical index of the pin on the ADC.
 |`chip_select`| string | **Required** | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the board's connection pin, wired to the ADC. |
-|`spi_bus` | string | Optional | The `name` of the SPI bus connecting the ADC and board. Required if your board must communicate with the ADC with the SPI protocol. |
+|`spi_bus` | string | Optional | The index of the SPI bus connecting the ADC and board. Required if your board must communicate with the ADC with the SPI protocol. |
```

Comment:
Unrelated to the change I made, but fixing a mistake in the docs: this is required, not optional. I'd also remove the second sentence ("Required if...protocol."). 

I don't believe we support any boards that have built-in analog readers. This has led to a bunch of confusion: the "analogs" section of the board config is really a config for an MCP3008 chip plugged into the board. I've been building consensus and laying the groundwork to split that into a separate component because it's a separate piece of hardware, but haven't written code for it yet.

Reply:
This sounds good to me.